.pre-commit-config.yaml

```diff
@@ -51,7 +51,7 @@ repos:
     rev: 6.1.1
     hooks:
       - id: pydocstyle
-        exclude: ^(gymnasium/envs/box2d)|(gymnasium/envs/classic_control)|(gymnasium/envs/mujoco)|(gymnasium/envs/toy_text)|(tests/envs)|(tests/spaces)|(tests/utils)|(tests/vector)|(tests/wrappers)|(docs/)
+        exclude: ^(gymnasium/envs/box2d)|(gymnasium/envs/classic_control)|(gymnasium/envs/mujoco)|(gymnasium/envs/toy_text)|(gymnasium/envs/jax_toy_text)|(tests/envs)|(tests/spaces)|(tests/utils)|(tests/vector)|(tests/wrappers)|(docs/)
```

Comment:
Is it possible to docstring all the jax toy text environments. 
We also need to come up with a better name than `jax_toy_text`, the new classic_control + mujoco environments are `phys2d` and `phys3d`.

Reply:
I think it wouldn't be unreasonable to call them 'tabular' since all toy_text environments are designed with tabular RL in mind.  And sounds good wrt adding the doc coverage.

Reply:
I actually really like the name `tabular`, lets use that for now

Reply:
ok pydocstyle is now enabled for `tabular` and it passes pre-commit